### PR TITLE
WIP distro: ec_transition: add initial support for ns5x_7xpu

### PR DESCRIPTION
**Pull request by TomaszAIR**
Signed-off-by: Tomasz Żyjewski <tomasz.zyjewski@3mdeb.com>

### CODE REVIEW

File: meta-dts-distro/recipes-dts/dts/files/ec_transition [hunk 1]
```diff
@@ -82,6 +82,16 @@ board_config() {
           BIOS_LINK="https://cloud.3mdeb.com/index.php/s/SKpqSNzfFNY7AbK/download"
           EC_LINK="https://cloud.3mdeb.com/index.php/s/GK2KbXaYprkCCWM/download"
           ;;
+        "NS5x_NS7xPU")
+          HAVE_EC="true"
+          NEED_EC_RESET="true"
+          COMPATIBLE_EC_FW_VERSION="TBD"
```
Comment (miczyg1) [inline meta-dts-distro/recipes-dts/dts/files/ec_transition:88]: What are we going to do with those TBDs? @TomaszAIR @mkopec ?

Comment (TomaszAIR) [reply meta-dts-distro/recipes-dts/dts/files/ec_transition:88]: @mkopec @miczyg1 In other cases we used cloud links so if we have EC there, please provide links and EC_FW_VERSION then we could just add them here. Do we have tools to flash vendor EC? We would need to test this ec_transition on NS5x_NS7xPU

**Comment by macpijan** [comment]
@mkopec @TomaszAIR we probably do not need this one anymore?

**Comment by TomaszAIR** [comment]
> @mkopec @TomaszAIR we probably do not need this one anymore?

Probably not. Will you also be able to confirm this, @mkopec ?

**Comment by mkopec** [comment]
Yes @macpijan @TomaszAIR I believe we don't need this anymore

**Comment by TomaszAIR** [comment]
@mkopec thanks, closing

**Pull request closed without merging**